### PR TITLE
Incremental import

### PR DESCRIPTION
**Pull request by andrewdotn**

### CODE REVIEW

**Comment by andrewdotn** [comment]
I’ll give this a few tweaks tomorrow morning, but this mostly covers it.

File: src/morphodict/lexicon/importer/testdata/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Test files for import process
+
+The data in here is **NOT linguistically correct**. It’s been tweaked in
+arbitrary ways to test what the importer would do if you made arbitrary changes
+to the input data.
```
Comment (dwhieb) [inline src/morphodict/lexicon/importer/testdata/README.md:5]: 👍🏼

File: src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py [hunk 2]
```diff
@@ -87,187 +41,67 @@ def __str__(self):
...
+class Command(BaseCommand):
+    help = """XXX No longer works
+
+            This command no longer works. It required that all wordforms be
+            instantiated in the database, even if they weren’t translatable, and
+            running it multiple times could result in multiple copies of
+            auto-definitions.
+            
+            Use importjsondict instead with the --translate-wordforms option
+            instead.
```
Comment (nienna73) [inline src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py:100]:
```suggestion
```

File: src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py [hunk 3]
```diff
@@ -87,187 +41,67 @@ def __str__(self):
...
-                                )
-                                for d in auto_definitions
-                            ],
-                        },
-                        ensure_ascii=False,
-                    )
-                    + "\n"
-                )
+    def add_arguments(self, parser: ArgumentParser):
+        pass
```
Comment (nienna73) [inline src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py:104]: You can likely remove this function entirely. I don't think it's necessary for the `BaseCommand` class.

File: src/morphodict/lexicon/importer/test_importer.py [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+        kw = SourceLanguageKeyword.objects.get(wordform=wf)
+        assert kw.text == "amisk2"
+
+    def test_indexing_of_unanalyzable_form(self):
+        """
+        Usually the FST handles finding the lemma for a source-language search;
+        that doen’t work when the FST doesn’t know about a term.
+
+        ‘maskwa’ is of course analyzable but in the test data for this test, no
+        ‘analysis is given in the importjson.
```
Comment (nienna73) [inline src/morphodict/lexicon/importer/test_importer.py:216]: There's either an extra or a missing ' in this sentence. (not a huge deal by any means)

File: src/morphodict/lexicon/importer/test_importer.py [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+    assert non_lemma.raw_analysis == [[], "nîmiw", ["+V", "+AI", "+Ind", "+X"]]
+
+    expected_defns = [["it is a dance, a time of dancing", "CW"]]
+    if translate_wordforms:
+        expected_defns.append(["someone dances", "🤖CW"])
+    assert definitions_match(non_lemma.definitions.all(), expected_defns)
+
+    assert {"time", "danc"}.issubset(
+        {t.text for t in TargetLanguageKeyword.objects.filter(wordform=non_lemma)}
+    )
```
Comment (nienna73) [inline src/morphodict/lexicon/importer/test_importer.py:530]: These tests are very thorough--I like it.

**Review by nienna73 — APPROVED** [review]
This looks great! I look forward to trying it out.

File: src/morphodict/lexicon/importer/test_importer.py (hunk 4 above)
Comment (andrewdotn) [reply src/morphodict/lexicon/importer/test_importer.py:216]: Thanks—that comes from a bug in the IntelliJ plugin I use to wrap text at 80 chars.

File: src/morphodict/lexicon/importer/test_importer.py (hunk 5 above)
Comment (andrewdotn) [reply src/morphodict/lexicon/importer/test_importer.py:530]: 😀

**Comment by andrewdotn** [comment]
Note: as part of this, I have removed several entries from the production database that had zero senses/definitions, e.g., https://itwewina.altlab.app/word/awîna@i/

**Pull request merged**